WTWSMS/common/religions/00_religions.txt

```diff
@@ -4775,6 +4801,19 @@ bosporan_group = {
 		can_grant_divorce = yes
 		can_call_crusade = yes
 		can_grant_invasion_cb = invasion
+
+		short_reign_opinion_year_mult = 0 # No short reign penalty
+
+		character_modifier = { 
+			fertility = 0.10
+			learning = 2
+			ai_honor = 1
+		}
+	
+		defensive_attrition = yes
+		independence_war_score_bonus = 50
+		attacking_same_religion_piety_loss = yes
+		has_heir_designation = yes
```

Comment:
Hypsistarian tribes seemed to be extremely decentralized.

I personally feel that Hypsistarians should be a faith that can be Reformed. Of course. "bosporan_faith" and "bosporan_faith_reformed" would be in the unique Bosporan Group.

The reason I say this is because there is no collusion between Hypsistarian faithful except for the family and the clans. They were a quiet, close-knit faith reknowned for their honor (I showed you the sources) that we have little information on.

A Reformed Hypsistarian faith would have a centralized, powerful religious head and have conjectural decisions that could be based on the Jewish Decisions.